cli/src/main/java/com/box/l10n/mojito/cli/command/CommandException.java

```diff
@@ -11,6 +11,8 @@
  */
 public class CommandException extends RuntimeException {
 
+  private boolean expected;
```

Comment:
`expected` name should be updated to `sendAlert`, or something to that affect. Default is `true` rather than `false`

Reply:
Done